cookiecutter/utils.py

```diff
@@ -12,9 +12,12 @@
 logger = logging.getLogger(__name__)
 
 
-def force_delete(func, path, exc_info):
+def force_delete(func, path, exc_info):  # pragma: no cover
```

Comment:
Please don't exclude this line. The CI have multiple os stage.

Reply:
This has been a dev leftover. I removed it now.